Drop-In/src/androidTest/java/com/braintreepayments/api/SupportedPaymentMethodsFragmentUITest.kt

```diff
@@ -25,10 +25,10 @@ import org.mockito.Mockito.mock
 class SupportedPaymentMethodsFragmentUITest {
 
     private val supportedPaymentMethods = listOf(
-            DropInPaymentMethodType.PAYPAL,
-            DropInPaymentMethodType.PAY_WITH_VENMO,
-            DropInPaymentMethodType.UNKNOWN,
-            DropInPaymentMethodType.GOOGLE_PAY
+            DropInPaymentMethod.PAYPAL,
+            DropInPaymentMethod.PAY_WITH_VENMO,
+            DropInPaymentMethod.UNKNOWN,
+            DropInPaymentMethod.GOOGLE_PAY
```

Comment:
`PaymentMethod` is a base class in the core SDK. It might make sense to move this enum into core in the next major version.